src/pruner_options.h

```diff
@@ -8,6 +8,11 @@
 
 namespace P4PRUNER {
 
+enum class BugType : char {
```

Comment:
This can just be a standard enum, I think we already have an enum for bug types called errortype?

Reply:
I was unsure if it would be better to create a separate one since we only expose 2 possibilities to the user, but I agree it makes sense to just reuse the same one here.